src/app/App.module.css

```
@@ -1,10 +1,8 @@
 .App {
   text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+  background-color: var(--primary-bg-color);
+  height: 100vh;
+  width: 100vh;
```

This should probably be vw, but I don't recommend setting it anywhere
```suggestion
  width: 100vw;
```